charts/prometheus-redis-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.3.4
+appVersion: 1.11.1
 description: Prometheus exporter for Redis metrics
 name: prometheus-redis-exporter
-version: 3.5.2
+version: 3.5.3
```

Comment:
as the image is some minor versions above the old version, it might introduce new features.
therefore i would raise the chart version to 3.6.0.

Reply:
@monotek - makes sense.  bumped the minor chart version since the application chart version also had a minor version change.  ready for re-review.